sharding/nodesSetup_test.go

```diff
@@ -1,10 +1,11 @@
-package sharding
+package sharding_test
 
 import (
 	"encoding/hex"
 	"testing"
 
 	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/sharding"
```

Comment:
exported vars are useless in tests.

Reply:
The tests were written in the sharding package and used the internal properties. I agree, unit tests should only test exported methods.

Reply:
reverted package to "sharding" and no need any more